docs/proposals/job_controller.md

```diff
@@ -0,0 +1,192 @@
+# Job Controller
+
+## Abstract
+A proposal for implementing a new controller - Job controller, which will be responsible
+for managing pods that require to run:
+* once on demand or at a specified point in time
+* repeatedly at a specified point in time.
+
+Several exiting issues were already created regarding that particular subject:
```

Comment:
I'm pretty sure that there are more than just these two issues, I will dig for some later.